tasks/java.yml

```diff
@@ -10,6 +10,12 @@
   yum: name={{ java }} state={{java_state}}
   when: ansible_os_family == 'RedHat'
 
+- name: correct java version selected
```

Comment:
Had issues with this on RHEL 7, the binary java8 does not exist

Reply:
i.e. it is already just java

Reply:
Amazon AMIs have different behaviour than RedHat. Made the change and tested on both Amazon AMI and redhat 7.4. Thanks @gregdurham for reviewing